src/CMakeLists.txt

```diff
@@ -123,24 +123,42 @@ set(GISH_H
 )
 
 if(WIN32)
-    list(append GISH_H video/glext.h)
+    list(APPEND GISH_H video/glext.h)
 endif(WIN32)
 
 if(APPLE)
     list(APPEND GISH_SRCS sdl/SDLMain.m)
     list(APPEND GISH_H sdl/SDLMain.h)
 endif(APPLE)
 
-add_executable(freegish ${GISH_SRCS} ${GISH_H})
+if(WIN32)
+    add_executable(freegish WIN32 ${GISH_SRCS} ${GISH_H})
```

Comment:
WIN32 is not needed. It'll build a Windows executable without this as well.

Reply:
Only if you build with Visuall Studio by hand. If you run `cmake --build .` to build WIN32 is needed here.

Reply:
Ah I wasn't aware of this method :+1: